power-supply/power_supply.cpp

```diff
@@ -704,7 +689,9 @@ void PowerSupply::updateInventory()
 
     assetProps.emplace(SN_PROP, sn);
     assetProps.emplace(PN_PROP, pn);
+#ifdef UCD90160_DEVICE_ACCESS
```

Comment:
UCD90160_DEVICE_ACCESS is not used to check UCD or mihawk. You need to create another macro.

Reply:
@alvintpwang I will fix it.